### PR TITLE
Moving the Jenkins image to VS2017 RTW.

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @jasonmalinowski, @dotnet/roslyn-infrastructure

### CODE REVIEW

**Comment by tannergooding** [comment]
test ci please

**Comment by tannergooding** [comment]
@jasonmalinowski, do we want to move to the RTW image in `dev15.0.x` or `dev15.1.x`?

File: netci.groovy [hunk 1]
```diff
@@ -71,7 +71,7 @@ set TMP=%TEMP%
 
       def triggerPhraseOnly = false
       def triggerPhraseExtra = ""
-      Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15-rc')
+      Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-vs15-0')
```
Comment (jasonmalinowski) [inline netci.groovy]: I would have assumed this was a VS2015 image. dev15.0 would probably be a better name.

**Comment by jasonmalinowski** [comment]
@tannergooding Yes, but let's flip this over first and monitor stability before we do.

File: netci.groovy (hunk 1 above)
Comment (jasonmalinowski) [inline netci.groovy]: (or call it vs2017-0 or something)

Comment (tannergooding) [inline netci.groovy]: Fixed.

**Comment by tannergooding** [comment]
test ci please

**Pull request merged**